components/molecule/accordion/src/index.js

```diff
@@ -0,0 +1,80 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import Tab from './Tab'
+
+const BASE_CLASS = 'sui-MoleculeAccordion'
+
+class MoleculeAccordion extends Component {
+  state = {
+    openTabs: []
+  }
+
+  onToggle = id => {
+    const {openTabs} = this.state
+    const {withAutoClose} = this.props
+    let newOpenTabs = []
+    if (withAutoClose) {
+      newOpenTabs[id] = openTabs[id] ? undefined : true
+    } else {
+      newOpenTabs = [...openTabs]
+      newOpenTabs[id] = newOpenTabs[id] ? undefined : true
+    }
+    this.setState({
+      openTabs: [...newOpenTabs]
+    })
+  }
+
+  render() {
+    const {openTabs} = this.state
+    const {children, icon, maxHeight, withTransition} = this.props
```

Comment:
[SUGGESTION] const {children, ...tabProps} = this.props then you can <Tab key={index} isOpen={!!openTabs[index]} {...tabProps}> so any new prop added to the tab will be automatically forwarded without the need of changing this code and so MoleculeAccordion will become agnostic about Tab props.